helm/chaos-mesh/values.yaml

```diff
@@ -21,7 +21,7 @@ controllerManager:
   replicaCount: 1
 
   image: pingcap/chaos-mesh:latest
-  imagePullPolicy: Always
+  imagePullPolicy: IfNotPresent
```

Comment:
Why change to `IfNotPresent`

Reply:
I think `always` is not necessary. Sometimes if the pod was restarted and we change some flags of it, which may cause the pod crashing.

Reply:
I think we should keep it set to `always`. Because our image is `latest`.  Unless we specify a particular version.

Reply:
Got it